DFC.ServiceTaxonomy.GraphVisualiser/Services/Neo4JToOwlGeneratorService1.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using DFC.ServiceTaxonomy.GraphVisualiser.Models;
+using DFC.ServiceTaxonomy.GraphVisualiser.Models.Owl;
+using Neo4j.Driver;
+
+namespace DFC.ServiceTaxonomy.GraphVisualiser.Services
+{
+    public class Neo4JToOwlGeneratorService : OwlDataGeneratorService, INeo4JToOwlGeneratorService
+    {
+        private long minNodeId;
+        private long minRelationshipId;
+
+        public OwlDataModel CreateOwlDataModels(long selectedNodeId, Dictionary<long, INode> nodes, HashSet<IRelationship> relationships, string prefLabel)
+        {
+            minNodeId = nodes.Keys.Min() - 1;
+            minRelationshipId = relationships.Count > 0 ? relationships.Min(r => r.Id) - 1 : 0;
+
+            TransformData(nodes, prefLabel);
+            TransformData(relationships);
+
+            var result = new OwlDataModel
+            {
+                Namespace = CreateNamespaces(),
+                Header = CreateHeader(),
+                Settings = CreateSettings(),
+                Class = nodeDataModels.Select(n => CreateClass(n, $"{selectedNodeId - minNodeId}")).ToList(),
+                ClassAttribute = nodeDataModels.Select(n => CreateClassAttribute(n)).ToList(),
```

Comment:
Should just be able to use the method group on these 3 lines, e.g. `Select(CreateClassAttribute)`

Reply:
done